challenges/solutions/all_first_color.js

```diff
@@ -18,3 +18,34 @@ for(var j=0; j<5; j++) {
   draw();
   moveRight();
 }
+
+
+/* 
+Note that the above solution actually moves off the Kyrel board.  Maybe that's OK.
+
+Sometimes in software development the instructions you're given are not 100% clear.  There are frequently edge-cases, or other possibilities that hadn't been thought of yet.
```

Comment:
You need a solution to discuss this.  It's not going to be clear on day 2 without that.